Quartz/Engine/Include/Quartz/Events/IEventListener.hpp

```diff
@@ -31,11 +36,11 @@ namespace qz
 	{
 		class IEventListener
 		{
-		public:
-			IEventListener() = default;
-			virtual ~IEventListener() = default;
+		  public:
```

Comment:
Why is this indented?

Reply:
Just a leftover from the default `.clang-format` and I didn't think it was obnoxious enough to change, do we want it back to aligning the access modifiers with the `class` declaration?
aka
```cpp
class Bob {
private:
    void dave ();
public:
    /// ...
};
```

Reply:
Yeah, imo I think aligning them with the class decl should be better - mainly so we dont have to deal with multiple different styles even if clang formats it right.